botcore/__init__.py

```diff
@@ -0,0 +1,9 @@
+from botcore import (
+    regex,
+)
+
+__all__ = [
+    regex,
+]
+
+__all__ = list(map(lambda module: module.__name__, __all__))
```

Comment:
What's the point of this?

Reply:
We expose certain modules, so that people using the package know what they can import, and more relevantly here for auto docs.

The reason that line is there is to convert to strings, the expected format. The reason I don't just hardcode the strings is is because they can be broken by refractings. Imports like this will throw an error instead of silently failing.

Reply:
Won't unknown modules in `__all__` raise a warning in linters though? I'm pretty sure I've seen linter warnings for that before.

Reply:
If they do, it's not set up in this project. Regardless, I think actual imports work better for automatic refactorings and guaranteed failures, what do you think?

Reply:
I'm still 50/50 on this, I'm not sure which linter complained about this though :(